addons/medical_damage/stringtable.xml

```diff
@@ -4,6 +4,7 @@
         <Container name="Settings">
             <Key ID="STR_ACE_Medical_damage_subCategory">
                 <English>Damage Threshold</English>
+                <German>Schaden</German>
```

Comment:
Is the English translation wrong here?

Reply:
Seems like "threshold" got lost in translation.

Reply:
Any suggestions for good translation?

Reply:
Schadensschwelle